tests/queries/0_stateless/01268_procfs_metrics.sh

```diff
@@ -8,14 +8,6 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 # shellcheck source=../shell_config.sh
 . "$CURDIR"/../shell_config.sh
 
-# NOTE: netlink taskstruct interface uses rounding to 1KB [1], so we cannot use ${BASH_SOURCE[0]}
-#
-#   [1]: https://elixir.bootlin.com/linux/v5.18-rc4/source/kernel/tsacct.c#L101
-tmp_path=$(mktemp "$CURDIR/01268_procfs_metrics.XXXXXX")
-trap 'rm -f $tmp_path' EXIT
-truncate -s1025 "$tmp_path"
-
-$CLICKHOUSE_LOCAL --profile-events-delay-ms=-1 --print-profile-events -q "SELECT * FROM file('$tmp_path', 'LineAsString') FORMAT Null" |& grep -m1 -F -o -e OSReadChars
```

Comment:
The proper fix is to add `--storage_file_read_method=pread`

The reason it started to fail is that the provider switched to netlink and so it does not have noisy `os_read_chars`, like the `procfs` provider has (it reads `/proc`).